lib/suspenders/app_builder.rb

```diff
@@ -388,6 +393,28 @@ def create_production_heroku_app(flags)
       run_heroku "create #{app_name} #{flags}", "production"
     end
 
+    def create_heroku_pipelines_config_file
+      template "app.json.erb", "app.json"
+    end
+
+    def create_heroku_pipeline
+      heroku_app_name = app_name.dasherize
+
+      pipelines_plugin = `heroku plugins | grep pipelines`
+      if pipelines_plugin.empty?
+        puts 'You need heroku pipelines plugin. Run: heroku plugins:install heroku-pipelines'
```

Comment:
Line is too long. [93/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.